content/solana-nft.md

```diff
@@ -0,0 +1,618 @@
+        "collection": "4cXtoUxF9C3nYxo31Hku6gFYDEcnidqSScbctfY17HZr"
+    },
+    "items": {
+        "0": {
+            "link": "https://arweave.net/nk7VSghTt9iP6cVYnVxo4SAN_t6RPxE6iESP-Z49s4s",
+            "imageLink": "https://arweave.net/8Y9i5mvF3u_3XHzSAfiDxt8pUubGB8HwXTTEnD4Kp7c?ext=png",
+            "name": "1",
+            "onChain": true,
+            "verifyRun": false
+        },
+        "1": {
+            "link": "https://arweave.net/LEbsEUVr-oMCgVkZlvSEmOoBRR7QcoFG-won6tZD16g",
+            "imageLink": "https://arweave.net/T78LQZsteVWK83g49wy-Lt3cig3w8N1kAXyZiU5Pxdg?ext=png",
+            "name": "2",
+            "onChain": true,
+            "verifyRun": false
+        },
+        "2": {
+            "link": "https://arweave.net/0D-klGK9bceLNm0Bxo4ua59RlLq1i0cM1A31Smsgvwo",
+            "imageLink": "https://arweave.net/GbPL0RTX_IEJvbcAEwqGI2nyzsT6IXRHDgk_o_fdwJQ?ext=png",
+            "name": "3",
+            "onChain": true,
+            "verifyRun": false
+        },
+        "3": {
+            "link": "https://arweave.net/hz0lrDqCvtVnvToHjmSCDlSF7feCcu8KX_zJDq_BBRc",
+            "imageLink": "https://arweave.net/mhxIIf-U4xwA1yFouxZx1eUmpMOr-7gz9a377fwbO8w?ext=png",
+            "name": "4",
+            "onChain": true,
+            "verifyRun": false
+        },
+        "4": {
+            "link": "https://arweave.net/MQsmrTW6BIufImiDjHY3py3dM1w8DdowKGtJ307MQW4",
+            "imageLink": "https://arweave.net/otRFxnPyLW_fmbUvskf4DE63n8dUurHt5Qm--gWWOdA?ext=png",
+            "name": "5",
+            "onChain": true,
+            "verifyRun": false
+        }
+    },
+    "env": "devnet",
+    "cacheName": "example"
+}
+```
+
+Next, let's verify our upload by running the `verify_upload` command.
+
+This verifies that each entry in the `devnet-example.json` file has been successfully uploaded and matches the URI stored on-chain.
+
+```sh
+ts-node metaplex/js/packages/cli/src/candy-machine-v2-cli.ts verify_upload \
+    -r, --rpc-url https://metaplex.devnet.rpcpool.com\
+    -k ./private-key.json \
+    -c example
+```
+
+The output should look something like this:
+
+![Gif of Metaplex Verify Upload](../assets/solana-nft-metaplex-verify-upload.gif)
+
+Now let's mint an NFT from our Candy Machine by running the `mint_one_token` command:
+
+```sh
+ts-node metaplex/js/packages/cli/src/candy-machine-v2-cli.ts mint_one_token \
+    -r, --rpc-url https://metaplex.devnet.rpcpool.com\
+    -k ./private-key.json \
+    -c example
+```
+
+![Gif of Metaplex Mint One](../assets/solana-nft-metaplex-mint-one.gif)
+
+You should now be able to see the newly minted NFT in the Phantom wallet.
+
+![Screenshot of Metaplex NFT](../assets/solana-nft-metaplex-nft.png)
+
+### 7. Candy Machine UI
+
+Now that we know our candy machine works, let's set up a frontend to mint our NFTs using the Candy Machine UI.
+
+Navigate to the candy-machine-ui folder using the following path: `/metaplex/js/packages/candy-machine-ui`.
```

Comment:
@jamesrp13 wrap the folder name in backticks, e.g. 

Navigate to the `candy-machine-ui` folder